### ISSUE TITLE
Turkish Language selection in settings not working

### ISSUE THREAD

**Issue by smntsn** [Brewtarget/brewtarget#670]
First of all I'm using this great software about 2 years and I LOVE IT :) THANK YOU :)
My problem is (also for 2x versions) Turkish language selection changes nothing (Interface still in English).
I fixed it by downloading Turkish translation>converting to .qm with QtLinguist and replacing it in installation folder(for 2x version).

But, now I can't do it. 

I don't know if supplied "bt_tr.qm" is not Turkish or setting page language selection is not working.

Thanks in advance. 

![Ekran görüntüsü 2022-10-19 174931](https://user-images.githubusercontent.com/13038415/196725333-395bd870-2c59-400a-91d6-79696f47efef.jpg)

**Comment by matty0ung** [comment]
OK, so I've fixed the build so that it is properly synchronising the "translations/*.ts" files (ie translation source files) with the translatable strings in the source code.  Once this fix is merged, you should be able to make useful edits to translations/bt_tr.ts.

I don't know whether generating the .qm file from QtLinguist and manually installing it is always guaranteed to work, but either way if you share your translations with us then we'd be very happy to incorporate them in the main build.

**Comment by matty0ung** [comment]
For now, I've put a "placeholder" translation in for "About Brewtarget" so it says "Turkish for about Brewtarget".  This means that, by showing the "About" dialog, we can convince ourselves that "display in Turkish" is picking up strings from the right place.

Once we have some Turkish translations in https://github.com/Brewtarget/brewtarget/blob/develop/translations/bt_tr.ts then we won't need that placeholder.

**Comment by matty0ung** [comment]
Sorry, jumped the gun a bit.  Wait for https://github.com/Brewtarget/brewtarget/pull/672 to be merged.

**Comment by matty0ung** [comment]
We'll do a 3.0.2 release soon, which will contain the fixes mentioned above.

**Comment by matty0ung** [comment]
The [3.0.2](https://github.com/Brewtarget/brewtarget/releases/tag/v3.0.2) release is now out, so I'm closing this ticket.  But please don't hesitate to re-open it (or raise another issue) if you hit problems with that new release.
